### DISCUSSION TITLE
zh-cn/manual/introduction/startup

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#140] [Comments]
# zh-cn/manual/introduction/startup

开始唤醒 自动启动模拟器，自动启动客户端，自动进入游戏。 自动启动模拟器需额外设置，详见启动设置，且需勾选连接设置内的 ADB 连接失败时尝试启动模拟器。 账号切换 仅支持切换至已登录的账号，使用登录名进行查找，请保证输入内容在所有已登录账号唯一。 例子： 官服：123****8901，可输入 123****8901、123、8901、3****8 B...

https://docs.maa.plus/zh-cn/manual/introduction/startup.html

**Comment by 2407135388** [comment]
账号切换不能正确使用
场景描述：b服有两个账号，一个四...，一个五...
无论是四切五还是五切四都不能实现，会正确退出账号，然后就会卡在登录界面，无法切换

**Reply by Rbqwow** [reply]
B服的切换有问题 界面大小缩放不一样
目前没找到办法修 开模拟器多开吧

**Reply by ABA2396** [reply]
现在b服好像只有720p还是1080p能切，试试看

**Comment by DLmaster361** [comment]
根据我的测试，B服当前在两个分辨率下都无法正确切换账号

**Reply by ABA2396** [reply]
之前有人成功过，1080p 下的 320dpi 还是多少

**Comment by lion575** [comment]
请问B服唤醒游戏每次概率出现的bilibili游戏政策隐私提醒的情况，他就会卡在那不动，并不会自动选择同意选项，能否解决，是需要做什么特别设置吗？

**Reply by DLmaster361** [reply]
MAA项目组之前好像说过不会做这个功能

**Reply by lion575** [reply]
大家都这样的话那就无所谓了，我还以为是我少设置了什么

**Comment by Citlalisdog** [comment]
如果我一个官服一个b服，能自动打完一个号切到另一个号的配置吗？还是要必须手点一下？

**Reply by DLmaster361** [reply]
官服到b服是要切换服务器的，MAA在这个过程中需要重启，想要在MAA里实现可能需要定时执行结合自定义配置选择。不过多账号的话可以考虑使用第三方插件AUTO_MAA（打广告中~）

**Comment by Minazuki-Hotaru** [comment]
官服的博士要注意一下，使用自动唤醒功能时，填写手机号中间的几位数字必须用“*”代替，也就是跟你登录界面显示的一模一样才行，不然MAA识别不了就会卡在登录界面：）

**Reply by Minazuki-Hotaru** [reply]
或者只输入号码的前三位也可以，如果你没有多个账号的话

**Reply by ABA2396** [reply]
没多个账号还填这个干嘛呢
鼠标放到那个问号上是有提示的

**Comment by linkingness909** [comment]
之前能够在不重启maa的情况下直接在运行设置里切换官服和b服启动，现在是不行了吗大佬们？还是要重新设置一下？

**Reply by ABA2396** [reply]
那是切换配置，你配置里怎么设置不是看你自己吗

**Reply by linkingness909** [reply]
您好大佬，可能是我表达的不清楚，是这样的，之前我是maa运行完b服的任务之后关闭mumu模拟器上的b服客户端，然后我再在运行设置里变更为官服，再点击执行任务开始，这时maa就直接能打开官服完成任务，而不会打开b服，好像是7周年更新之后我就发现这样不行了，上一次运行完b服后我切换成官服，maa依然打开b服，不知道是为什么。

**Reply by ABA2396** [reply]
之前有个 pr 写的有问题，v6.9.1 修

**Reply by linkingness909** [reply]
好的，非常感谢大佬的回复

**Comment by Citlalisdog** [comment]


**Reply by ABA2396** [reply]
把邮件回复关了，不然就 ban 了

**Comment by sillya1a2** [comment]
断线后自动重连功能在新版本删掉了吗？

**Reply by ABA2396** [reply]
断线后自动重连只在刷理智功能支持

**Comment by Minazuki-Hotaru** [comment]
为什呢你们的邮件回复会抄送给我



---Original---
From: ***@***.***&gt;
Date: Sat, May 9, 2026 23:49 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [MaaAssistantArknights/maa-website] zh-cn/manual/introduction/startup (Discussion #140)




 
断线后自动重连只在刷理智功能支持
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by ABA2396** [reply]
因为你订阅了消息，不行要的话应该点
<img width="962" height="1416" alt="image" src="https://github.com/user-attachments/assets/c3a3c90c-4b40-43e7-b5b4-415a6d5b9d20" />

**Comment by Godenglow** [comment]
B 服：
1. bilibili 游戏政策隐私提醒弹窗：可以用 AUTO-MAS 添加 MAA，并在「设置 → 功能设置 → 托管 Bilibili 游戏隐私政策」中设为「是」（默认值为「否」）。

2. 账号识别（切换）问题：找到 MAA 安装目录下的 maa/debug/asst.log，把这个文件发给 AI，告诉它你的账号名，问你的账号被识别成了什么、哪些片段识别不出错，再把那段不出错的片段填入。


我的日志里id识别是这样的
翻车记录排序：g（→9/q/丢，必错）、l（偶发丢）、大写首字母（W→w 3 次）。通用高危名单：g/9/q、l/1/I、O/0/o、大写开头
